Terminal.Gui/View/Layout/ViewLayout.cs

```diff
@@ -209,7 +209,7 @@ void ThicknessChangedHandler (object sender, EventArgs e)
 			Padding.ThicknessChanged -= ThicknessChangedHandler;
 			Padding.Dispose ();
 		}
-		Padding = new Frame () { Id = "Padding", Thickness = new Thickness (0) };
+		Padding = new Frame () { Thickness = new Thickness (0) };
```

Comment:
In the future if `Padding` has a different behavior than `Frame` then you will to sub-class it.